hare/algorithm.go

```diff
@@ -255,39 +256,44 @@ func (proc *ConsensusProcess) onEarlyMessage(m *Msg) {
 		return
 	}
 
-	if m.HareMessage == nil {
-		proc.Error("onEarlyMessage called with nil message")
+	if m.XDRMessage == nil {
+		proc.Error("onEarlyMessage called with nil Message")
+		return
+	}
+
+	if m.Message == nil {
```

Comment:
should you process with concensus process after receiving a null message

Reply:
Shouldn't be possible, prefer an error